avicena/app/__init__.py

```diff
@@ -0,0 +1,10 @@
+import logging.config
+import yaml
+
+with open('config/log_config.yaml', 'r') as cfg_file:
+    log_config = yaml.load(cfg_file, Loader=yaml.FullLoader)
+    log_config['disable_existing_loggers'] = False
+    logging.config.dictConfig(log_config)
```

Comment:
move into a function such as `init_logging` rather than running as a side effect of importing `app`